### ISSUE TITLE
More verbosity

### ISSUE THREAD

**Issue by ocharles** [andrewchambers/terraform-provider-nix#13]
Is it possible to get the output of `nix-build` into the `terraform apply` output somehow?

**Comment by andrewchambers** [comment]
I think this may be a limitation of the terraform. I remember looking into it, I agree it would be really useful. Maybe I missed something.

**Comment by rehno-lindeque** [comment]
Related I think: https://github.com/hashicorp/terraform/issues/20784

**Comment by andrewchambers** [comment]
Yeah, I tried to use TF_LOG=DEBUG myself, but bugs in the google cloud provider started printing binary output which ruined my terminal sessions, really annoying oversight by hashicorp.
